components/DefinedTerm.js

```diff
@@ -104,9 +104,11 @@ const UnderlinedTerm = styled.span`
  * Underlines the given word and show a tooltip with the definition when focused
  * or hovered. Both the term and the definition are translated.
  */
-const DefinedTerm = ({ intl, term, textTransform, fontSize, children, color }) => {
+const DefinedTerm = ({ intl, term, textTransform, fontSize, children, color, extraTooltipContent }) => {
   return (
-    <StyledTooltip content={() => intl.formatMessage(TranslatedDefinitions[term], TranslationParams[term])}>
+    <StyledTooltip
+      content={() => [intl.formatMessage(TranslatedDefinitions[term], TranslationParams[term]), extraTooltipContent]}
```

Comment:
This is generating a React warning `Each child in a list should have a unique "key" prop.`. I suggest:

```suggestion
      content={() => (
        <React.Fragment>
          {intl.formatMessage(TranslatedDefinitions[term], TranslationParams[term])}
          {extraTooltipContent}
        </React.Fragment>
      )}
```